lib/main.coffee

```diff
@@ -2,6 +2,17 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+# This function returns true if version v2 is greater than v1
+isGreaterVersion = (v1, v2) ->
+  v1Parts = v1.split('.')
+  v2Parts = v2.split('.')
+  minLength = Math.min(v1Parts.length, v2Parts.length)
+  if minLength > 0
+    for idx in [0..minLength - 1]
+      diff = Number(v1Parts[idx]) - Number(v2Parts[idx])
+      return diff < 0 unless diff is 0
+  return (v1Parts.length - v2Parts.length) < 0
```

Comment:
again why don't you do `v1Parts.length < v2Parts.length`? Way easier to understand.